scripts/eda_dev.sh

```diff
@@ -94,8 +94,45 @@ start-events-services() {
     log-debug "docker-compose -p ansible-events -f tools/docker/docker-compose.yml up -d postgres"
     docker-compose -p ansible-events -f tools/docker/docker-compose.yml up -d postgres
 
-    until [ "$(docker inspect -f '{{.State.Health.Status}}' eda-postgres)" == "healthy" ]; do
+    #
+    # Podman versions previous to v4 use a different string variation for health check.
+    #
+    # Reference:
+    #      https://github.com/containers/podman/issues/11645
+    #
+    # Default is to use the Docker key value if both Podman and Docker are installed.
+    #
+    local heath_check=".State.Health.Status"
+
+    if which docker &> /dev/null; then
```

Comment:
This won't work because the "docker" binary is required even if podman is installed. Podman must be checked first.

Reply:
Is this due to our requirement of Docker-compose?

Reply:
now check to see if podman is installed first.